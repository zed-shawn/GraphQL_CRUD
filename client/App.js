import React from 'react';
import {ApolloClient, InMemoryCache, ApolloProvider} from '@apollo/client';
import 'cross-fetch/polyfill';
import Screen from './src/Screens/Screen';

const App = () => {
  const client = new ApolloClient({
    uri: 'http://10.0.2.2:3001/graphql',
    cache: new InMemoryCache(),
  });
  return (
    <ApolloProvider client={client}>
      <Screen />
    </ApolloProvider>
  );
};

export default App;
