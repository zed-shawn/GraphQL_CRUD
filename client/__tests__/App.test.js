import 'react-native';
import React from 'react';
import App from '../App';

import renderer from 'react-test-renderer';

test('Renders the app correctly', () => {
  const snap = renderer.create(<App />).toJSON();

  expect(snap).toMatchSnapshot();
});
