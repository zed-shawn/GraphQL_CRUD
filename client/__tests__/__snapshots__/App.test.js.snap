// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the app correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <Text
    style={
      Object {
        "flex": 1,
        "fontSize": 30,
        "fontWeight": "bold",
        "textAlignVertical": "center",
      }
    }
  >
    GraphQL + ERN Client
  </Text>
  <View
    style={
      Object {
        "flex": 3,
        "justifyContent": "space-evenly",
      }
    }
  >
    <TextInput
      onChangeText={[Function]}
      placeholder="Enter Name"
      style={
        Object {
          "borderRadius": 10,
          "borderWidth": 1,
        }
      }
    />
    <TextInput
      onChangeText={[Function]}
      placeholder="Enter Username"
      style={
        Object {
          "borderRadius": 10,
          "borderWidth": 1,
        }
      }
    />
    <TextInput
      onChangeText={[Function]}
      placeholder="Enter Password"
      secureTextEntry={true}
      style={
        Object {
          "borderRadius": 10,
          "borderWidth": 1,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      accessibilityRole="button"
      accessible={true}
      collapsable={false}
      focusable={true}
      nativeID="animatedComponent"
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "black",
              },
            ]
          }
        >
          Submit
        </Text>
      </View>
    </View>
  </View>
</View>
`;
